node/node.go

```diff
@@ -273,7 +273,7 @@ func (n *Node) currentRole() api.NodeRole {
 
 // configVxlanUDPPort sets vxlan port in libnetwork
 func configVxlanUDPPort(ctx context.Context, vxlanUDPPort uint32) {
-	if err := overlayutils.ConfigVxlanUDPPort(vxlanUDPPort); err != nil {
+	if err := overlayutils.ConfigVXLANUDPPort(vxlanUDPPort); err != nil {
```

Comment:
wasn't there a swarm type that had to be renamed as well? let me double check (looking at my phone right now)